draft-packages/tile/KaizenDraft/Tile/components/GenericTile.tsx

```diff
@@ -1,58 +1,56 @@
-import React, { useState, MouseEvent } from "react"
+import React, { useState, MouseEvent, HTMLAttributes } from "react"
+import classnames from "classnames"
+import { OverrideClassName } from "@kaizen/component-base"
 import { Box } from "@kaizen/component-library"
-import { Heading, Paragraph, AllowedHeadingTags } from "@kaizen/typography"
-import { IconButton } from "@kaizen/button"
-import classNames from "classnames"
-
 import informationIcon from "@kaizen/component-library/icons/information.icon.svg"
 import arrowBackwardIcon from "@kaizen/component-library/icons/arrow-backward.icon.svg"
+import { IconButton } from "@kaizen/button"
+import { Heading, Paragraph, AllowedHeadingTags } from "@kaizen/typography"
 import Action from "./Action"
 import styles from "./GenericTile.scss"
 
 export interface TileAction {
-  readonly label: string
-  readonly onClick?: (e: MouseEvent) => void
-  readonly href?: string
-  readonly icon?: React.SVGAttributes<SVGSymbolElement>
-  readonly automationId?: string
-  readonly newTabAndIUnderstandTheAccessibilityImplications?: boolean
+  label: string
+  onClick?: (e: MouseEvent) => void
+  href?: string
+  icon?: React.SVGAttributes<SVGSymbolElement>
+  automationId?: string
+  newTabAndIUnderstandTheAccessibilityImplications?: boolean
 }
 
 export interface TileInformation {
-  readonly text: string
-  readonly primaryAction?: TileAction
-  readonly secondaryAction?: TileAction
+  text: string
+  primaryAction?: TileAction
+  secondaryAction?: TileAction
 }
 
-export interface GenericTileProps {
-  readonly title: React.ReactNode
-  readonly titleTag?: AllowedHeadingTags
-  readonly metadata?: string
-  readonly children?: React.ReactNode
-  readonly information?: TileInformation | React.ReactNode
-  readonly mood?:
+export interface GenericTileProps
+  extends OverrideClassName<Omit<HTMLAttributes<HTMLDivElement>, "title">> {
+  children?: React.ReactNode
+  title: React.ReactNode
+  titleTag?: AllowedHeadingTags
+  metadata?: string
+  information?: TileInformation | React.ReactNode
+  mood?:
```

Comment:
Now that we've got something like component-base we could potentially house our type for moods type in there. Not necessarily in this fix, but something worth considering 🤔 💭

Reply:
Certainly something we could consider later :) The `component-base` is meant to house shared things after all.